lib/wufoo.js

```diff
@@ -19,53 +19,58 @@ module.exports = Wufoo;
 function Wufoo(account, apiKey) {
    this.account = account;
    this.url = "https://"+account+".wufoo.com/api/v3/";
-   this.apiKey = apiKey;   
+   this.apiKey = apiKey;
 }
 
 
 // get all the forms for this account/subdomain.
 Wufoo.prototype.getForms = function(fn) {
-   this._getObjects("forms", fn, "Forms", Form);
+   this._getObjects("forms", null, fn, "Forms", Form);
 }
 
 // get the form given the id/hash.
 Wufoo.prototype.getForm = function(id, fn) {
-   this._getObjects("forms/" + id, fn, "Forms", Form, true);
+   this._getObjects("forms/" + id, null, fn, "Forms", Form, true);
 }
 
 // get all the form entries given the form id/hash.
 Wufoo.prototype.getFormEntries = function(id, fn) {
-   this._getObjects("forms/" + id + "/entries", fn, "Entries", GenericEntity);
+   this._getObjects("forms/" + id + "/entries", null, fn, "Entries", GenericEntity);
+}
+
+// get all the form entries given the form id/hash.
+Wufoo.prototype.getFormEntriesById = function(id, entryId, fn) {
```

Comment:
@igoldny Since you are essentially wanting to obtain a specific `Entry` wouldn't this return one object?

i.e. `this._getObjects("forms/" + id + "/entries", null, fn, "Entries", GenericEntity)[0]`

There will always be only an array of length 1.

And if that's the case this function may be better named as `getFormEntriesById`

Thoughts?

Reply:
this PR is 2 years old, I don't even remember doing it.
you probably know better then I am now.
you want me to change it to "getFormEntriesById"?